drbd/defaults.yaml

```diff
@@ -1,15 +1,13 @@
 drbd:
-  install_packages: true
-  with_ha: true
+  install_packages: false
+  with_ha: false
+  promotion: "drbd-node1"
```

Comment:
I wouldn't put a default promotion node here. I think it's a bit strange. And actually, I think It would be better if the execution fails if you have not put the value, as most of the times this value will be different.

Reply:
Indeed, it make sense. The parameters more likely different per settings.
I would remove this including `res_template` and `resource` section.